Rol de manager

alejandropiuryp opened the pull request:
Se ha creado la entidad de Manager y modificado el diagrama UML, también se ha añadido ejemplos de manager a la base de datos.

alejandropiuryp committed:
Rol de manager

Se ha creado la entidad de Manager y modificado el diagrama UML, también se ha añadido ejemplos de manager a la base de datos.

manpercar1 reviewed (changes requested):
En la clase ManagerUserAccountUpdate.java falta una parte del código en el método findOne, lo cual da lugar a un error.

alejandropiuryp committed:
Eliminadas las clases sobrantes

Se han eliminado las clases por ahora sobrante de la entidad manager: su repositorio y su UpdateService, que además estaban carentes de código

manpercar1 committed:
Merge branch 'develop' into feature/Managers

manpercar1 reviewed (approved):
Todo correcto
